tests/displayAllGamesTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+require_once 'src/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class displayAllGamesTest extends TestCase {
+    public function testDisplayGamesMalformedInputs(): void
+    {
+        $games = "This should be an array";
+        $this->expectException(TypeError::class);
+        displayAllGames($games);
+    }
```

Comment:
Surely you can write a success test as well?

Reply:
Done :) Please can you review?